File: FizzBuzz/FizzBuzz.py
for num in range(1, 100):
    
    if (num % 3 == 0 and num % 5 == 0):
        print('FizzBuzz')

    elif (num % 3 == 0):
        print('Fizz')

    elif (num % 5 == 0):
        print('Buzz')

    else:
        print(num)

File: 3-Desafio_FizzBuzz/fizzBuzz.js
/*
Link do desafio:
https://dojopuzzles.com/problems/fizzbuzz/
*/

var num = 100;

function imprima(num) {
for(var i = 1; i <= num; i++) {
    i % 3 == 0 && i % 5 == 0 ? console.log("FizzBuzz") : false
    i % 3 == 0 && i % 5 != 0 ? console.log("Fizz") : false
    i % 3 != 0 && i % 5 == 0 ? console.log("Buzz") : false
    i % 3 != 0 && i % 5 != 0 ? console.log(i) : false
 }
}

imprima(num);
